tensorrt_llm/_torch/modules/fused_moe.py

```diff
@@ -565,6 +569,12 @@ def forward_chunk(
         assert token_final_scales.dtype == torch.float32
         assert token_selected_experts.dtype == torch.int32
 
+        if self.apply_router_weight_on_input:
+            assert self.routing_method.top_k == 1, "Current walkaround only supports top-1 routing"
+            x = x * token_final_scales.to(x.dtype)
+            # TODO: remove this once we have correct fusedmoe kernel ready
+            token_final_scales = torch.ones_like(token_final_scales)
```

Comment:
```suggestion
            token_final_scales = None
```
Can you do a quick mmlu run to verify this?